src/Stripe.net/Entities/StripePlan.cs

```diff
@@ -35,10 +35,16 @@ public class StripePlan : StripeEntityWithId
         [JsonProperty("name")]
         public string Name { get; set; }
```

Comment:
Shouldn't we remove `Name` completely?

Reply:
Wouldn't that prevent existing API integrations from upgrading their bindings?

Reply:
I don't entirely follow. Anyone who gets that new version of the code will already have to change their integration since it's a new API version and we've added/moved/removed multiple properties.

Name does not exist on the new API version, only Nickname does so `Name` should go.

This will all be behind a major version

Reply:
Right, I'm talking the other way. Do merchants ever update their bindings independent of their API version? If not, then I can remove it. But if they do, then updating bindings to this version would eliminate their ability to access `Name`

Reply:
@pantera-stripe stripe-dotnet enforces an API version for the entire library. There is no way to "choose" an API version. Every user of stripe-dotnet is locked to the API version that the version of the library they use has been built for.

In another part of your PR you are now enforcing `2018-02-06` so every developer who upgrades their integration to that new version of stripe-dotnet will use that version on every API request. So `Name` will never exist or be set.

Reply:
Yeah. Totally overlooked that. Will remove.